rambo/cli.py

```diff
@@ -93,8 +93,11 @@ def cli(ctx, cwd, tmpdir_path, vagrant_cwd, vagrant_dotfile_path):
     app.set_init_vars(cwd, tmpdir_path)
     app.set_vagrant_vars(vagrant_cwd, vagrant_dotfile_path)
 
-    app.write_to_log('\nNEW CMD')
-    app.write_to_log(' '.join(sys.argv))
+    utils.write_to_log('\nNEW CMD')
```

Comment:
If we are going to do logging we need to use a propper python logging module. No need to roll ur own.